### PR TITLE
Passport application Refactor

### LINKED ISSUE #14: Enabling Testing Application Via Passport App
### Problem

Currently, the user needs to test the application, via calling the API. Needs to fix the issues in the Passport Application to run along with the Passport App by Calling the endpoint.

`http://localhost:9080/public/graphql`

### Proposed Solution

1. An Express.js Application (serving the built file of Passport Application Frontend).
2. The Express backend will perform the token exchange.
3. And When the user presses the Get Data button in the Passport Application. It will hit the Backend and Backend will act as a proxy, along with providing the required token parameters obtained from the IDP.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by mushrafmim**
## Summary
Provide a brief description of what this PR accomplishes and why these changes are needed.

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made
- List the main changes made in this PR
- Be specific about what was modified, added, or removed
- Include any architectural changes

## Testing
- [x] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [ ] All existing tests pass

## Checklist
- [x] My code follows the project's style guidelines
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Related Issues
Closes #14

### CODE REVIEW

File: members/die/applications/online-passport-app/server/src/middleware/auth.ts [hunk 1]
```diff
@@ -35,15 +36,20 @@ class TokenManager {
     try {
       const credentials = Buffer.from(`${this.clientId}:${this.clientSecret}`).toString('base64');
 
+      const httpsAgent = new https.Agent({
+        rejectUnauthorized: false, // 👈 trust self-signed certs
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply members/die/applications/online-passport-app/server/src/middleware/auth.ts]: This is only for development environment.

File: ndx/.env [hunk 2]
```diff
@@ -111,7 +111,7 @@ OE_SCHEMA_PATH=./schema.graphql
...
 # =============================================================================
 IDP_ORG_NAME=lankasoftwarefoundation
-IDP_BASE_URL=https://localhost:9443
-IDP_ISSUER=https://localhost:9443/oauth2/token
+IDP_BASE_URL=https://wso2is:9443
+IDP_ISSUER=https://wso2is:9443/oauth2/token
 IDP_AUDIENCE=Mpjt5VUqDPL8iVByyFcMDregz6Ea
-IDP_JWKS_URL=https://localhost:9443/oauth2/jwks
\ No newline at end of file
+IDP_JWKS_URL=http://wso2is:9763/oauth2/jwks
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply ndx/.env]: Since, this is local development, this is irrelevant

File: init.sh [hunk 3]
```diff
@@ -390,131 +312,238 @@ fi
...
+print_success "M2M Application Credentials"
+print_success "=========================================="
+print_info "Application Name: Passport Application"
+print_info "Client ID:        $M2M_CLIENT_ID"
+print_info "Client Secret:    $M2M_CLIENT_SECRET"
+print_success "=========================================="
+echo ""
+
+# Define the Consent Portal client ID
+PORTAL_CLIENT_ID="Mpjt5VUqDPL8iVByyFcMDregz6Ea"
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply init.sh]: Need to be hard coded, since the PORTAL service is also need to be up on system initialization
